spec.html

```diff
@@ -2813,10 +2813,10 @@ <h1>Object Internal Methods and Internal Slots</h1>
             For the internal methods listed in <emu-xref href="#table-essential-internal-methods"></emu-xref>, the object uses those defined in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>.
           </li>
           <li>
-            If the object has a [[Call]] internal method, it uses the one defined in <emu-xref href="#sec-ecmascript-function-objects-call-thisargument-argumentslist"></emu-xref>.
+            If the object has a [[Call]] internal method, it uses the one defined in either <emu-xref href="#sec-ecmascript-function-objects-call-thisargument-argumentslist"></emu-xref> or <emu-xref href="#sec-built-in-function-objects-call-thisargument-argumentslist"></emu-xref>.
```

Comment:
This grammar is a bit weird. "It uses the one defined in either X or Y" sounds like there is one thing which is defined in one of two places, as opposed to there being two things where either choice is acceptable.

```suggestion
            If the object has a [[Call]] internal method, it uses either the one defined in <emu-xref href="#sec-ecmascript-function-objects-call-thisargument-argumentslist"></emu-xref> or the one defined in <emu-xref href="#sec-built-in-function-objects-call-thisargument-argumentslist"></emu-xref>.
```